ansible/library/announce_routes.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
```

Comment:
Will release branches use the same sonic-mgmt container that is built with Python 3 based on Ubuntu 24.04 ? If so would the changes applied here have an impact for release branch testing?

Reply:
In this PR testing, the sonic-mgmt container is still the old one. This PR aims to test the changes won't impact current PR testing. So, it won't impact other release branch testing.

Reply:
Why make this change? Just for the purpose of testing it by PR test itself?
If so, the justification is not good enough.

Reply:
If the purpose is to try something, make the PR a draft and do not merge. And after testing is done, it would be better to close the draft PR.

Reply:
Hi, @wangxin , this PR needs to merge after review. As we will upgrade sonic-mgmt docker image to ubuntu 24.04, we should modify the test framework to support it. For this, as the default python path in ubuntu 24.04 is `/usr/bin/python`, we can't find python under path `/usr/bin/env python`. So we make this modification.